1.11/installing/ent/cloud/aws/index.md

```diff
@@ -21,4 +21,13 @@ Choose the advanced templates if you want to customize the AMI, VPC, or Instance
 
 * Highly customizable
 * Composable, for example you can deploy multiple agent pools to a single cluster
-* More setup work is required
\ No newline at end of file
+* More setup work is required
+
+[message type="warning"] DC/OS CloudFormation templates are intended for reference only and are not recommended for production use due to the following limitations:
+
+- CloudFormation does not allow for coordinated zero-downtime in-place updates within Auto Scaling groups.
+- CloudFormation does not allow for automated zero-downtime replacement of Auto Scaling groups.
+- Replacing DC/OS agent nodes requires manual data migration of local storage volumes for stateful services.
+- Updates of DC/OS on AWS CloudFormation has not been automated, validated, or documented.
```

Comment:
Got a problem with verb noun agreement here. "updates" and "has" don't go together. 

Either change "has" to "have" or change "updates of" to "updating".

Change needs to be made on all copies.